packages/core/content-type-builder/admin/src/translations/en.json

```diff
@@ -133,14 +133,13 @@
   "form.button.select-component": "Select a component",
   "form.button.single-type.description": "Best for single instance like about us, homepage, etc.",
   "form.contentType.divider.draft-publish": "Draft/Publish",
+  "form.button.collection-type.name": "Collection Type",
+  "form.button.single-type.name": "Single Type",
   "from": "from",
   "listView.headerLayout.description": "Build the data architecture of your content",
-  "menu.section.components.name.plural": "Components",
-  "menu.section.components.name.singular": "Component",
-  "menu.section.models.name.plural": "Collection Types",
-  "menu.section.models.name.singular": "Collection Type",
-  "menu.section.single-types.name.plural": "Single Types",
-  "menu.section.single-types.name.singular": "Single Type",
+  "menu.section.components.name": "Components",
+  "menu.section.models.name": "Collection Types",
```

Comment:
I think we need to delete the corresponding keys in the other language can you run the cleaning script?

Reply:
Sure, doing this rn. Just wanted to do keep the diff readable :)

Reply:
@soupette done!